docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -308,101 +309,174 @@ Here is a sample of this configuration file:
-just for human readability, only the actual account IDs listed here matter. If you specify ``public=public`` here, the AGFIs are shared publicly, regardless of any other entires that are present.
+``agfis_to_share`` with when calling the manager's ``shareagfi`` command. You
+should specify names in the form ``usersname: AWSACCTID``. The left-hand-side is
+just for human readability, only the actual account IDs listed here matter. If you specify ``public: public`` here, the AGFIs are shared publicly, regardless of any other entires that are present.
+
+.. _config-build-farm:
+
+``config_build_farm.yaml``
+--------------------------
+
+Here is a sample of this configuration file:
+
+.. include:: /../deploy/sample-backup-configs/sample_config_build_farm.yaml
+   :code: yaml
+
+Below, we outline each section and parameter in detail.
+
+Build farm configuration sections, e.g. ``awesome_build_farm``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this file, you can specify as many build farm configuration sections as you want,
+each with a header like ``awesome_build_farm`` (i.e. a nice, short name
+you made up). Such a section must contain the following fields:
+
+``build_farm_type``
+"""""""""""""""""""""
+
+This field maps the build farm configuration to a specific build farm class name
+(or in other words, a build farm definition). By default, build farm classes can
+be found in :gh-file-ref:`deploy/buildtools/buildfarm.py`. However, you can specify
+your own custom build farm classes by adding your python file to the ``PYTHONPATH``.
+For example, to use the ``AWSEC2`` build farm class, you would write ``build_farm_type: AWSEC2``.
+
+``args``
+"""""""""""""""""""""
+
+This section specifies all arguments needed for the specific ``build_farm_type`` used.
+For a list of arguments needed for a specific build farm class, users should refer to
+the ``_parse_args`` function in the build farm class given by ``build_farm_type``.
+
+Mainline build farm configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are two mainline build farm configurations provided (with their corresponding build farm classes in :gh-file-ref:`deploy/buildtools/buildfarm.py`).
+
+``ec2_build_farm (AWSEC2)``
+"""""""""""""""""""""""""""
+
+.. literalinclude:: /../deploy/sample-backup-configs/sample_config_build_farm.yaml
+   :language: yaml
+   :start-after: DOCREF START: EC2 Build Farm
+   :end-before: DOCREF END: EC2 Build Farm
+
+This build farm configuration provides the default EC2 build farm functionality
+of FireSim 1.13.x and earlier. For each AGFI (FireSim AWS EC2 F1 FPGA Image) being
+built, a new EC2 instance (build host) will be spawned according to the build farm configuration
+arguments. Its ``build_farm_type`` is set as the ``AWSEC2`` build farm class.
+
+The ``instance_type`` argument defines the type of instance that the build will run on.
+Generally, running on a ``z1d.2xlarge`` is sufficient. In our experience, using more
```

Comment:
```suggestion
Generally, running on a ``z1d.2xlarge`` is sufficient. For very large designs, using a z1d.3xlarge may be required to handle Vivado's memory footprint.`
```
You'll have to clean up the next line.

Reply:
It's still worth saying higher frequency instances may not help that much.

Reply:
Help in speeding up builds correct?